Mathlib/Algebra/Group/Semiconj.lean

```diff
@@ -56,33 +53,98 @@ theorem mul_right (h : SemiconjBy a x y) (h' : SemiconjBy a x' y') :
   unfold SemiconjBy
   -- TODO this could be done using `assoc_rw` if/when this is ported to mathlib4
   rw [←mul_assoc, h.eq, mul_assoc, h'.eq, ←mul_assoc]
+#align semiconj_by.mul_right SemiconjBy.mul_right
 
-/-- If both `a` and `b` semiconjugate `x` to `y`, then so does `a * b`. -/
-@[to_additive "If both `a` and `b` semiconjugate `x` to `y`, then so does `a + b`."]
+/-- If `b` semiconjugates `x` to `y` and `a` semiconjugates `y` to `z`, then `a * b`
+semiconjugates `x` to `z`. -/
+@[to_additive "If `b` semiconjugates `x` to `y` and `a` semiconjugates `y` to `z`, then `a + b`
+semiconjugates `x` to `z`."]
 theorem mul_left (ha : SemiconjBy a y z) (hb : SemiconjBy b x y) : SemiconjBy (a * b) x z := by
   unfold SemiconjBy
   rw [mul_assoc, hb.eq, ←mul_assoc, ha.eq, mul_assoc]
+#align semiconj_by.mul_left SemiconjBy.mul_left
```

Comment:
There should be a separate #align for the additive version. I guess we need a linter for this.